cmd/integration_test/build/buildspecs/test-eks-a-cli.yml

```diff
@@ -37,7 +39,7 @@ env:
     EKSA_GITHUB_TOKEN: "github-eks-anywhere-flux-bot:github-token"
     T_GITHUB_USER: "github-eks-anywhere-flux-bot:github-user"
     T_GIT_REPOSITORY: "github-eks-anywhere-flux-bot:github-repository"
-    T_GIT_SSH_AUTHORIZED_KEY: "github-eks-anywhere-flux-bot:github-ssh-key"
+    T_GIT_SSH_PRIVATE_KEY: "github-eks-anywhere-flux-bot:github-ssh-key"
```

Comment:
```suggestion
    T_GIT_SSH_PRIVATE_KEY: "github-eks-anywhere-flux-bot:github-ssh-private-key"
```

Reply:
we're actually now opting to download this from a file in s3 instead of passing it as an env var